port_drayage_plugin/src/port_drayage_plugin.cpp

```diff
@@ -36,13 +36,17 @@ namespace port_drayage_plugin
 
         ros::Publisher outbound_mob_op = _nh->advertise<cav_msgs::MobilityOperation>("outgoing_mobility_operation", 5);
         _outbound_mobility_operations_publisher = std::make_shared<ros::Publisher>(outbound_mob_op);
+
+        _set_active_route_client = _nh->serviceClient<cav_srvs::SetActiveRoute>("/guidance/set_active_route");
+
         PortDrayageWorker pdw{
             cmv_id,
             cargo_id,
             "HOST_ID",
```

Comment:
Is this a placeholder string?

Reply:
Yes it is. I can change this to a config parameter.